src/components/Button.js

```diff
@@ -233,6 +233,10 @@ class Button extends Component {
         return (
             <Pressable
                 onPress={(e) => {
+                    if (e.type === 'click') {
```

Comment:
Sorry, let's also check if e.type exists so

if (e.type && e.type === 'click')

Same with below

Reply:
Handled.

Reply:
The way I mentioned is preferred

Reply:
Why though? If `e` is undefined, what you've suggested will throw an error.

Reply:
fair enough, let's just check if e exists then

Reply:
How about `if (e?.type === 'click')` ?

Reply:
`if (e.type && e.type === 'click')` is exactly same as `e.type === 'click`. Not sure what you're trying to do here.

Reply:
if e && e.type === 'click'

Reply:
Fixed.